README.md

```diff
@@ -13,15 +13,15 @@ backend support for GitHub, Enterprise Github, and more! See [here](https://vime
 curl -sS http://gushphp.org/installer | php
 ```
 
-* Logo explanation is best presented from this passage in Psalms 78*:
+*Logo explanation is best presented from this passage in Psalms 78*:
 
 > True, he struck the rock, and water gushed out, streams flowed abundantly, but can he also give us bread?
 > Can he supply meat for his people?” When the Lord heard them, he was furious; his fire broke out against
 > Jacob, and his wrath rose against Israel, for they did not believe in God or trust in his deliverance.
-
-There first thread of blood and the following are water gushing out of a rock, connecting the Old Testament
-prophecy fulfillment in the New Testament at the cross when Jesus was opened on his side and gushed out
-water and blood.
+> 
+> There first thread of blood and the following are water gushing out of a rock, connecting the Old Testament
```

Comment:
this makes my explanation Sacred Scripture so it is wrong. Good though you are pondering the meaning :+1: